<?xml version="1.0"?>

<launch>
  <arg name="input/tl_state" default="traffic_light_states"/>
  <arg name="input/vector_map" default="/map/vector_map"/>
  <arg name="output/traffic_light" default="traffic_light_states/markers"/>

  <node pkg="traffic_light_visualization" type="traffic_light_map_visualizer_node" name="traffic_light_map_visualizer">
    <remap from="~input/tl_state" to="$(arg input/tl_state)"/>
    <remap from="~input/vector_map" to="$(arg input/vector_map)"/>
    <remap from="~output/traffic_light" to="$(arg output/traffic_light)"/>
  </node>
</launch>
